src/babel.js

```diff
@@ -275,7 +275,7 @@ export default function ({types: t}) {
             ])
           }
 
-          state.jsxId = hash(state.styles.map(s => s[1]).join(''))
+          state.jsxId = hash(state.styles.map(s => s[1].source || s[1]).join(''))
```

Comment:
that might blow up all the test and require us to refactor them, in that case maybe we can stick with your solution